src/utils/hooks/useFetchCustomerInstitution.ts

```diff
@@ -0,0 +1,52 @@
+import { useEffect, useState } from 'react';
+import Axios from 'axios';
+import { useDispatch } from 'react-redux';
+import { CustomerInstitution, emptyCustomerInstitution } from '../../types/customerInstitution.types';
+import { getCustomerInstitution } from '../../api/customerInstitutionsApi';
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+
+export const useFetchCustomerInstitution = (
+  identifier: string,
+  editMode: boolean
+): [CustomerInstitution, boolean, (customerInstitution: CustomerInstitution) => void] => {
+  const dispatch = useDispatch();
+  const [isLoading, setIsLoading] = useState(true);
+  const [customerInstitution, setCustomerInstitution] = useState<CustomerInstitution>(emptyCustomerInstitution);
```

Comment:
litt misvisende å fortsatt bruke `emptyCustomerInstitution` (i stedet for ingenting) imo, siden dette nå er en fetchCustomerInstitution og ikke lenger en setInitialValues

Reply:
ja, du har rett i det 👍